frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { OverviewItem, Status } from '@console/shared';
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
+import { ListGroup } from 'patternfly-react';
+import { Grid, GridItem } from '@patternfly/react-core';
+import {
+  SidebarSectionHeading,
+  ResourceLink,
+  resourcePath,
+} from '@console/internal/components/utils';
+import { PipelineRunModel, PipelineModel } from '@console/dev-console/src/models';
+import { Link } from 'react-router-dom';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import { pipelineRunFilterReducer } from '../../../utils/pipeline-filter-reducer';
+import TriggerLastRunButton from './TriggerLastRunButton';
+
+const noOfPipelineRunsToshow = 3;
```

Comment:
Optional change...
```suggestion
const MAX_VISIBLE = 3;
```
Could go with a shorter name so it doesn't read like a phrase.

Reply:
fixed in https://github.com/openshift/console/pull/3056/commits/8e27cbfc8c376c202bd2471c788bf0401b8dcb9d